client/src/screens/CombineResultsScreen.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react'
+import fileDownload from 'js-file-download'
+import styled from 'styled-components'
+
+import ConverterClient from '../lib/ConverterClient'
+
+import { InputEventFunction } from '../config/types'
+
+import Button from '../components/Button'
+import LinkButton from '../components/LinkButton'
+import routerPaths from '../routerPaths'
+import FileInputButton from '../components/FileInputButton'
+import HorizontalRule from '../components/HorizontalRule'
+import Prose from '../components/Prose'
+import Loading from '../components/Loading'
+import NavigationScreen from '../components/NavigationScreen'
+
+const Loaded = styled.p`
+  line-height: 2.5rem;
+  color: rgb(0, 128, 0);
+  &::before {
+    content: '✓ ';
+  }
+`
+
+const CombineResultsScreen = () => {
+  const [client] = useState(new ConverterClient('election'))
```

Comment:
Why `useState` if you are not defining a `setClient` second array element?
```suggestion
  const client = new ConverterClient('election')
```

Reply:
lol, because I was copying from `UnconfiguredScreen` and there's no good reason.